### PR TITLE
add a few tests with docker-compose

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @bfirsh @aanand you might want to take a look

**Comment by vieux** [comment]
@aluzzardi in the future we might add `docker-compose` to ou test-env image

File: test/integration/test_runner.sh [hunk 1]
```diff
@@ -12,11 +12,15 @@ function execute() {
...
 
 # Tests to run. Defaults to all.
-TESTS=${@:-. discovery api mesos/api}
+TESTS=${@:-. compose discovery api mesos/api mesos/compose}
 
 # Generate a temporary binary for the tests.
 export SWARM_BINARY=`mktemp`
 
+# Download docker-compose
+execute time curl -L --silent https://github.com/docker/compose/releases/download/1.4.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```
Comment (chanwit) [inline test/integration/test_runner.sh]: Could you make `docker-compose` version configurable?
Like this:
`https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose`

I think it's better for regression across different version of `docker-compose` too.

**Comment by chanwit** [comment]
@vieux a small comment on docker-compose versioning. Others than that LGTM.

I really wanted to see this integration :):):)

**Comment by vieux** [comment]
@chanwit done, PTAL

**Comment by chanwit** [comment]
LGTM

File: test/integration/compose/compose_helpers.bash [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+load ../helpers
+
+function docker-compose_swarm() {
+	 DOCKER_HOST=${SWARM_HOSTS[0]} docker-compose "$@"
+}
```
Comment (abronan) [inline test/integration/compose/compose_helpers.bash]: woops should add a new line :ghost:

File: test/integration/testdata/compose/simple-resource.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+service1:
+  image: busybox
+  command: sleep 100
+  cpu_shares: 1
+
+service2:
+  image: busybox
+  command: sleep 100
+  cpu_shares: 1
```
Comment (abronan) [inline test/integration/testdata/compose/simple-resource.yml]: same here

**Comment by abronan** [comment]
This is super cool :+1: Just a few issues with useless extra-lines and new line characters missing at the end.

**Comment by bfirsh** [comment]
Niiice! :+1:

Doesn't need to be in this PR, but just as a heads up: These things have been a bit wobbly with Swarm, so probably worth testing:

- Building images (containers not getting scheduled to where built images are, different versions of built images in the Swarm, etc)
- Recreating containers (changing a Compose file then doing "up" again)

**Comment by abronan** [comment]
LGTM, thanks for addressing the comments :)

**Pull request merged**